src/main/java/com/rsmart/kuali/kfs/module/ld/businessobject/lookup/CurrentFundsLookupableHelperServiceImpl.java

```diff
@@ -1,339 +0,0 @@
-/*
- * Copyright 2011 The Kuali Foundation.
- * 
- * Licensed under the Educational Community License, Version 1.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- * http://www.opensource.org/licenses/ecl1.php
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.rsmart.kuali.kfs.module.ld.businessobject.lookup;
-
-import org.apache.commons.collections4.IteratorUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import org.kuali.kfs.gl.Constant;
-import org.kuali.kfs.gl.businessobject.TransientBalanceInquiryAttributes;
-import org.kuali.kfs.integration.ld.LaborLedgerObject;
-import org.kuali.kfs.integration.ld.businessobject.inquiry.AbstractPositionDataDetailsInquirableImpl;
-import org.kuali.kfs.module.ld.LaborKeyConstants;
-import org.kuali.kfs.module.ld.businessobject.AccountStatusCurrentFunds;
-import org.kuali.kfs.module.ld.businessobject.July1PositionFunding;
-import org.kuali.kfs.module.ld.businessobject.LaborObject;
-import org.kuali.kfs.module.ld.businessobject.LedgerBalance;
-import org.kuali.kfs.module.ld.businessobject.inquiry.AbstractLaborInquirableImpl;
-import org.kuali.kfs.module.ld.businessobject.inquiry.PositionDataDetailsInquirableImpl;
-import org.kuali.kfs.module.ld.dataaccess.LaborDao;
-import org.kuali.kfs.module.ld.service.LaborInquiryOptionsService;
-import org.kuali.kfs.module.ld.service.LaborLedgerBalanceService;
-import org.kuali.kfs.sys.KFSConstants;
-import org.kuali.kfs.sys.KFSPropertyConstants;
-import org.kuali.kfs.sys.ObjectUtil;
-import org.kuali.rice.core.api.util.type.KualiDecimal;
-import org.kuali.kfs.kns.lookup.AbstractLookupableHelperServiceImpl;
-import org.kuali.kfs.kns.lookup.HtmlData;
-import org.kuali.kfs.kns.lookup.HtmlData.AnchorHtmlData;
-import org.kuali.kfs.kns.web.ui.Row;
-import org.kuali.rice.krad.bo.BusinessObject;
-import org.kuali.kfs.krad.lookup.CollectionIncomplete;
-import org.kuali.kfs.krad.service.BusinessObjectService;
-import org.kuali.kfs.krad.util.BeanPropertyComparator;
-import org.kuali.kfs.krad.util.GlobalVariables;
-
-import com.rsmart.kuali.kfs.module.ld.businessobject.inquiry.CurrentFundsInquirableImpl;
-
-public class CurrentFundsLookupableHelperServiceImpl extends AbstractLookupableHelperServiceImpl {
```

Comment:
I could not find a reference to this rsmart version of CurrentFundsLookupableHelperServiceImpl. Should we still keep it?

Reply:
You're right, I don't see this rsmart version of the lookupable being referenced in our Spring beans, along with some of the other rsmart-related classes as well.

Was this class simply being deleted because it looked like an unneeded overlay of a base Rice class? If that's the case, I would suggest keeping it in place for now, and we can use a separate ticket to clean up these rsmart classes. I'll create a new user story for that cleanup.

Reply:
There were a lot of changes to be brought in and since the class wasn't used I have decided it was less work to just remove it. I'd like to create a separate jira just for the other rsmart classes.

Reply:
In that case, I'm fine with this class being removed as part of this PR. Thanks for the input.

Also, I've created KFSPTS-16909 to clean up the rest of these rsmart classes.

Reply:
Whoops, I see that you already created KFSPTS-16910 for this rsmart class cleanup. Sorry for the duplication.

Feel free to resolve one of those tickets as a duplicate of the other.

Reply:
Thank you. Closed mine as duplicate and linked.